operator/test/e2e/suite_test.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"context"
+	"fmt"
+	klfc "github.com/keptn/lifecycle-toolkit/operator/api/v1alpha1"
```

Comment:
```suggestion
	klcv1alpha1 "github.com/keptn/lifecycle-toolkit/operator/api/v1alpha1"
```

Reply:
I would expect `kltv1alpha1`

Reply:
@thisthat should I profit from this and rename the import everywhere as kltv1alpha1?

Reply:
however thinking about that here we import the controller not the whole toolkit... so maybe klcv1alpha1 is the way to go